src/ducks/ui/listevisning-selectors.ts

```diff
@@ -1,6 +1,6 @@
-import {AppState} from '../../reducer';
-import {Kolonne, ListevisningState, OversiktType} from './listevisning';
-import {AktiviteterValg, FiltreringAktiviteterValg} from '../filtrering';
+import { AppState } from "../../reducer";
```

Comment:
It seems you are using some local linting rules and not the ones in .prettierrc.json, could you check?

Reply:
Fixed, I can't find configuration for prettier..had to call it from terminal on each commit :(